connectors/aws-cert/src/main/java/com/vmware/connectors/aws/cert/AwsCertController.java

```diff
@@ -258,7 +258,8 @@ private Card makeCard(
 
         CardAction.Builder dismissAction = CardAction.Builder
                 .dismissAction()
-                .setLabel(cardTextAccessor.getActionLabel("dismiss"));
+                .setLabel(cardTextAccessor.getActionLabel("dismiss"))
+                .setCompletedLabel(cardTextAccessor.getActionCompletedLabel("dismiss"));
```

Comment:
In my opinion, just use the default (even if we can't specify locale).  They will never really see this completed label.  The whole point of the dismiss button is to make the card go away.

Reply:
Modified the ```completedLabel``` to have normal action label. Changed the test cases accordingly.